filebeat/docs/modules/threatintel.asciidoc

```diff
@@ -0,0 +1,312 @@
+////
+This file is generated! See scripts/docs_collector.py
+////
+
+[[filebeat-module-threatintel]]
+[role="xpack"]
+
+:modulename: threatintel
+:has-dashboards: false
+
+
+== Threat Intel module
+beta[]
+
+This module is a collection of different threat intelligence sources.
```

Comment:
I think it would be good to provide context around how this data can be utilized with the Elastic Stack (threat indicator rules and maybe enrichment processors?).